### ISSUE TITLE
[FEATURE] Export RETURN_VALUE_ERROR_CODES

### ISSUE THREAD

**Issue by skibitsky** [enzoferey/ethers-error-parser#33]
Please add `RETURN_VALUE_ERROR_CODES` to the package exports. 

I would like to make a switch statement around error codes to handle errors nicely, but it isn't possible because my app code cannot access `RETURN_VALUE_ERROR_CODES`:
![screenshot](https://user-images.githubusercontent.com/15127974/192143022-ec558b7f-6121-44c8-9fcf-ea6dd80eb64c.png)

**Comment by enzoferey** [comment]
Hey @skibitsky ! 

The idea I had in mind when designing this part of the package is that you would be using string literals to match the types. TypeScript will allow you to narrow the cases and have and clean switch statement like:

```ts
const ethersError = getParsedEthersError(error);

switch(ethersError.errorCode) {
      case "EXECUTION_REVERTED": { ... }
      case "INSUFFICIENT_FUNDS_FOR_GAS": { ... }
      .....
      default: { ... }
}
```

I recognize this may lead to typos or issues for people not using TypeScript though, so I'm fine exporting the object for convenience 👍🏻

**Comment by enzoferey** [comment]
@skibitsky available on version `0.2.2` 🚀
